microsoft-edge/devtools-guide-chromium/javascript/consume-source-maps-from-azure.md

```diff
@@ -64,43 +104,85 @@ To generate a PAT in Azure DevOps:
 
 To learn more about PAT, see [Use personal access tokens](/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate).
 
+With your new PAT generated, configure DevTools:
+
+1. In DevTools, click **Settings** (![Settings icon](./consume-source-maps-from-azure-images/settings-gear-icon-light-theme.png)) > **Symbol Server**.
+
+1. Click **Add Symbol Server connection** to start creating a new connection.
+
+1. In the **Authorization mode** dropdown, select **Personal Access Token**.
+
+1. In the **Azure DevOps organization** text box, enter the Azure DevOps organization where you created the PAT.
+
+1. In the **Personal access token** text box, paste your personal access token (PAT).
+
+1. Click **Add** to create the connection.
+
+   ![The Symbol Server configuration screen in DevTools settings, showing the fields required to create a new PAT connection](./consume-source-maps-from-azure-images/create-new-pat-connection.png)
+
+1. Click the **Close** (![Close settings icon](./consume-source-maps-from-azure-images/close-settings-icon.png)) button  in the upper right to close the **Settings** panel, and then click the **Reload DevTools** button.
+
 
 <!-- ====================================================================== -->
-## Step 2: Configure DevTools
+## Step 3: Retrieve original code in DevTools
 
-DevTools now needs to be configured with the personal access token (PAT) to successfully retrieve the source maps.
+After the above setup steps, when you use DevTools to work on a build of your website for which symbols have been published, you can now see your original source code, instead of the transformed code.
```

Comment:
```suggestion
After doing the above setup steps, when you use DevTools to work on a build of your website for which symbols have been published, you can now see your original source code, instead of the transformed code.
```
* "doing" - more explicit